docs/elements/mysql/tips.md

```diff
@@ -10,11 +10,15 @@ parent: Back to Element Guides
 order: 50
 ---
 
-## Tips
+# Tips
 
-### Why can't I connect my database?
+The Tips section includes more information about the element including FAQs, endpoint limitations, and general tips.
 
-Check your config file to make sure the port and bind-address have been set.
+{% include callout.html content="<strong>On this page</strong></br><a href=#general>General</a></br><a href=#events>Events</a></br><a href=#events>Events</a></br><a href=#documentation>Documentation</a></br>" type="info" %}
+
+## General
+
+* If you are having trouble connecting to your database, check your config file to make sure the port and bind-address have been set.
 
 Example Config:
```

Comment:
In the last sentence, we say "The db.host value would equal = 127.0.0.1:3306." But it doesn't really tell us why that's important or what db.host is. I recommend changing it to: "In this case when you authenticate an element instance, use `127.0.0.1:3306` as the value for `db.host`."